components/list.css

```diff
@@ -0,0 +1,43 @@
+.list-container {
+  width: 18rem;
+  background-color: var(--primary-color);
+  margin: 1rem;
+  text-align: center;
+}
+
+.norm-list-item {
+  border: 1px solid var(--cta-color);
+  width: 100%;
+  text-align: center;
+  padding: 10px;
+  font-size: 1.2rem;
+  color: var(--cta-color);
+}
+
+.norm-list-item > div {
+  color: var(--primary-color);
+  background-color: var(--secondary-color);
+  border-radius: 50%;
```

Comment:
If using this value frequently then you can use this as a variable named as --bd-rd-circle:50% and use it accordingly.

Reply:
nice idea to add utility thank you for suggestion